source/patterns/@aws-solutions-constructs/aws-apigateway-sagemakerendpoint/lib/index.ts

```diff
@@ -170,6 +178,7 @@ export class ApiGatewayToSageMakerEndpoint extends Construct {
       apiResource,
       requestValidator,
       requestTemplate: props.requestMappingTemplate,
+      additionalRequestTemplates: props.additionalRequestTemplates,
```

Comment:
No checks for validity of props values?

Reply:
Added a new check + test to make sure application/json content-type is passed in the requestTemplate property only.